spec/integration/cli/new/hanami_head_spec.rb

```diff
@@ -16,9 +16,9 @@
         expect('Gemfile').to have_file_content(%r{gem 'hanami-view',        require: false, git: 'https://github.com/hanami/view.git',        branch: 'develop'})
         expect('Gemfile').to have_file_content(%r{gem 'hanami-helpers',     require: false, git: 'https://github.com/hanami/helpers.git',     branch: 'develop'})
         expect('Gemfile').to have_file_content(%r{gem 'hanami-mailer',      require: false, git: 'https://github.com/hanami/mailer.git',      branch: 'develop'})
-        expect('Gemfile').to have_file_content(%r{gem 'hanami-cli',         require: false, git: 'https://github.com/hanami/cli.git',         branch: 'develop'})
         expect('Gemfile').to have_file_content(%r{gem 'hanami-assets',      require: false, git: 'https://github.com/hanami/assets.git',      branch: 'develop'})
         expect('Gemfile').to have_file_content(%r{gem 'hanami-model',       require: false, git: 'https://github.com/hanami/model.git',       branch: 'develop'})
+        expect('Gemfile').to have_file_content(%r{gem 'dry-cli',            require: false, git: 'https://github.com/dry-rb/dry-cli.git',     branch: 'master'})
```

Comment:
Because since now on we'll depend on `dry-cli` versions from Rubygems, there is no need to add it to the `Gemfile`.

Reply:
It looks like you removed `hanami-cli` from `lib/hanami/cli/commands/new/Gemfile.erb`, but did not add `dry-cli`, which is **correct**. I believe that this spec will fail.

Reply:
yep, I've already passed that moment. Thanks. I have another error at another place now :)